index.html

```diff
@@ -122,7 +122,7 @@ <h4 class="card__title">Skills</h4>
                     <h2 class="heading-2">I'm always interested in hearing about new projects, so if you'd like to chat, please get in touch.</h2>
                 </div>
                 <div class="contact__form">
-                    <form action="https://formspree.io/f/mnqyjlky"
+                    <form action=""
```

Comment:
- [ ] **[Optional]** I think it would be better to add a value to the action attribute of your form because it specifies the URL of the page to which the form data will be submitted. 👍 Kindly try to Collect data by using the [Formspree service](https://formspree.io/html). to make it professional 😎😎